.travis.yml

```diff
@@ -77,3 +77,4 @@ script:
   - php core/scripts/run-tests.sh --suppress-deprecations --verbose --color --concurrency 4 --types "PHPUnit-Functional" --php `which php` --url http://localhost:8080 "workbench_access"
   - php core/scripts/run-tests.sh --suppress-deprecations --verbose --color --concurrency 4 --types "PHPUnit-Kernel" --php `which php` --url http://localhost:8080 "workbench_access"
   - php core/scripts/run-tests.sh --suppress-deprecations --verbose --color --concurrency 4 --types "PHPUnit-Unit" --php `which php` --url http://localhost:8080 "workbench_access"
+  - vendor/bin/phpunit -c core "modules/contrib/workbench_access/workbench_access_protect/tests/Functional/TaxonomySchemeUITest.php"
```

Comment:
I would use the same method as above, which means annotating the tests, I believe.

Reply:
Changed to the same method, still failing.